posthog/api/team.py

```diff
@@ -334,17 +332,16 @@ def update(self, instance: Team, validated_data: Dict[str, Any]) -> Team:
         return updated_team
 
 
-class TeamViewSet(viewsets.ModelViewSet):
+class TeamViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     """
     Projects for the current organization.
     """
 
     serializer_class = TeamSerializer
     queryset = Team.objects.all().select_related("organization")
-    permission_classes = [permissions.IsAuthenticated, PremiumMultiProjectPermissions]
+    permission_classes = [IsAuthenticated, PremiumMultiProjectPermissions]
```

Comment:
`IsAuthenticated` is in by default, right? I'd rather remove it here to not cause confusion like thinking "wait, did I forget it elsewhere"?

Reply:
Not quite - again Team is a weird one that has its own overridden get_permissions.
I still want the gains from the base class but the permissions are different here

Reply:
I see, then it makes sense.